src/tokenizer/main.py

```diff
@@ -145,6 +153,11 @@ def quote(s: str) -> str:
             backslashes and double quotes escaped with a backslash """
         return '"' + s.replace("\\", "\\\\").replace('"', '\\"') + '"'
 
+    def spanquote(l: list) -> str:
+        """ Return the list l as a string within double quotes """
+        stringlist = [str(x) for x in l]
+        return '"' + "-".join(stringlist) + '"'
```

Comment:
This could be more compact as `return f'"{"-".join(str(x) for x in l)}"'` (then the stringlist assignment above is not needed)